actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -88,13 +92,15 @@
 
     private CompletableFuture<Void> hostingActive = new Task<>();
 
-    private int maxLocalAddressCacheCount = 10_000;
-
     private NodeSelectorExtension nodeSelector;
 
-    public Hosting()
+    public Hosting(final int maxLocalAddressCacheCount, final long defaultActorTTL)
```

Comment:
Agreed. I think it's reasonable to still have it but have it default to 0 which is unlimited. 
That way, if it becomes a concern for any use case it's overridable, but in the default case we don't limit it and instead rely on the access time.